ulohy_07/1D_piskvorky/util.py

```diff
@@ -0,0 +1,11 @@
+def tah(pole, symbol, zvolena_pozice):
+    """Vrátí herní pole se zaznamenaným tahem hráče"""
+    if not 0 <= zvolena_pozice < len(pole):
+        raise IndexError("Pozice je mimo rozsah herního pole. Zvol prosím pozici 0 - 19.")
```

Comment:
Co by se stalo, kdyby zde nebylo `raise IndexError(...)` a přesto někdo zadal číslo mimo rozsah?

Reply:
No kdyby to tam nebylo, tak by to mělo vyhodit chybu `IndexError`, protože se jedná o neexistující index stringu (pojmenováno dle taháku k této lekci). To `raise IndexError` tu mám kvůli tomu negativnímu testu. Více v komentáři k negativním testům.